app/create/index.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+var Angular = require('angular');
```

Comment:
Nit: We usually separate 3rd party library imports from internal module imports with a newline. [Example](https://github.com/lob/dashboard/blob/master/app/dashboard/templates/create/index.js#L3-L6)